### PR TITLE
Add jekyll to view examples

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Add example jekyll site to mirror vanilla-framework

## QA

- Run `rm -rf node_modules`
- Run `npm i`
- Run `bundle install`
- Run `gulp jekyll`
- Example site should be at: `http://localhost:4001`

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+##
+## This config for the the jekyll instance at /jekyll which is used to test Examples
+## during development. To view, run: `jekyll serve`
+##
+name: Vanilla framework - Examples
```
Comment (anthonydillon) [inline _config.yml]: Vanilla brochure theme

File: _site/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8"/>
+        <title> Home |  Vanilla framework - Examples</title>
```
Comment (anthonydillon) [inline _site/index.html]: Vanilla brochure theme

File: _site/index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    </head>
+
+    <body>
+        <div class="row">
+            <div class="col-12">
+                 <!-- Header -->
+<div class="p-strip">
+    <div class="row">
+      <div class="12-col">
+        <h1>Vanilla framework - Examples</h1>
```
Comment (anthonydillon) [inline _site/index.html]: Vanilla brochure theme

File: _site/patterns/lists/split/index.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8"/>
+        <title> Lists / Split |  Vanilla framework - Examples</title>
```
Comment (anthonydillon) [inline _site/patterns/lists/split/index.html]: Vanilla brochure theme

File: _site/index.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```
Comment (anthonydillon) [inline _site/index.html]: I thought the `_site` directory was git ignored?

File: jekyll/_layouts/default.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+<html>
+    <head>
+        <meta charset="utf-8"/>
+        <title>{% if page.title %} {{ page.title }} | {% endif %} {{ site.name }}</title>
+        <link rel="stylesheet" type="text/css" href="{{site.baseurl}}/css/build.css" />
+    </head>
+
+    <body>
+        <div class="row">
+            <div class="col-12">
```
Comment (anthonydillon) [inline jekyll/_layouts/default.html]: Could you please remove the `row` and `col-12`as it's being displayed when I grab the markup snippet for the docs.

File: jekyll/_patterns/lists/split.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Lists / Split
+category: _base
```
Comment (anthonydillon) [inline jekyll/_patterns/lists/split.html]: This is a pattern not a base style.

**Comment by anthonydillon** [comment]
I think we should merge master into develop to keep the history of commits?

**Comment by barrymcgee** [comment]
Updated to reflect Robin's structure change.

File: _jekyll/_layouts/default.html [hunk 8]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8"/>
+        <title>{% if page.title %} {{ page.title }} | {% endif %} {{ site.name }}</title>
+        {% if site.env == 'development' %}
+          <link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/build/css/build.css" />
+        {% else %}
+          <link rel="stylesheet" type="text/css" href="https://assets.ubuntu.com/latest-redirects/vanilla-framework.css" />
```
Comment (barrymcgee) [inline _jekyll/_layouts/default.html]: @nottrobin We'll need to setup this mechanism for production theme styles also.

Comment (nottrobin) [inline _jekyll/_layouts/default.html]: Huh?

Comment (nottrobin) [inline _jekyll/_layouts/default.html]: Oh you mean the latest redirect? Sure

Comment (barrymcgee) [inline _jekyll/_layouts/default.html]: Yes.

**Comment by anthonydillon** [comment]
No QA. Will QA once I get in this morning unless someone else gets to it.

**Pull request merged**